### PR TITLE
Ol3kjak050319

### PR DESCRIPTION
**Pull request by ol3kjak**

### CODE REVIEW

File: fast-read/src/assets/copy.js [hunk 1]
```diff
@@ -47,7 +47,54 @@ const copy = {
...
+      rows: 'Rzędy',
+      stop: 'Stop',
+      start: 'Start',
+      restart: 'Restart',
+      rewind: 'Przewijanie',
+      reset: 'Reset',
+    },
+    faq: [
+      {
+        question: 'Pytsnie 1',
```
Comment (michalmietlinski) [inline fast-read/src/assets/copy.js]: Change to "pytanie"

File: fast-read/src/assets/copy.js [hunk 2]
```diff
@@ -47,7 +47,54 @@ const copy = {
...
+        answear: 'Odpowiedź 1',
+      },
+      {
+        question: 'Pytsnie 2',
```
Comment (michalmietlinski) [inline fast-read/src/assets/copy.js]: Change to "pytanie"

File: fast-read/src/assets/copy.js (hunk 1 above)
Comment (ol3kjak) [inline fast-read/src/assets/copy.js]:
```suggestion
        question: 'Pytanie 1',
```

File: fast-read/src/assets/copy.js (hunk 2 above)
Comment (ol3kjak) [inline fast-read/src/assets/copy.js]:
```suggestion
        question: 'Pytanie 2',
```

**Pull request merged**